### DISCUSSION TITLE
Security headers shipped in the boilerplate

### DISCUSSION THREAD

**Discussion by f-ricci** [laravel/framework#41146] [Ideas]
Hi,

I have recently run into some troubles because our website was missing the header [x-frame-options](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options)

As the framework already builds in some security measures like CSRF, SQL injections, among others, I thought it would be nice if it also shipped with some defaults for Content Security Policy.

I would like to be able to provide more to this topic but unfortunately I don't have enough expertise in this topic.

**Comment by crynobone** [comment]
It's already available via https://github.com/laravel/framework/blob/b6f47323da0f0052680d337fe49ffdfe12692ad6/src/Illuminate/Http/Middleware/FrameGuard.php#L7
